web/src/components/ChatConversation.tsx

```diff
@@ -29,10 +33,15 @@ type ChatConversationProps = {
   className?: string
 }
 
+const TYPING_INDICATOR_TIMEOUT = 60000
+
 const ChatConversation = ({ messages, hasError, className }: ChatConversationProps): ReactElement => {
   const { t } = useTranslation('chat')
   const [messagesCount, setMessagesCount] = useState(0)
+  const [showTypingIndicator, setShowTypingIndicator] = useState(false)
   const messagesEndRef = useRef<HTMLDivElement>(null)
+  const showIndicatorOfTyping =
+    messages[messages.length - 1]?.userIsAuthor || messages[messages.length - 2]?.userIsAuthor
```

Comment:
This means that if the message before the last one is a user it will show the 3 dot.
Shouldn't be shown just if the last message is coming from a user?

Reply:
Fair point and I think you are right. Seems like I was a little tired already when I made my review, sorry :D Probably we would need a condition in the following way:
Show the indicator if either
- the last message was sent by the user OR
- there is only one message yet that was NOT sent by the user (as that is the automatic answer)